src/perf_counters.cc

```diff
@@ -33,7 +33,7 @@ const bool PerfCounters::kSupported = true;
 bool PerfCounters::Initialize() { return pfm_initialize() == PFM_SUCCESS; }
 
 PerfCounters PerfCounters::Create(
-    const std::vector<std::string>& counter_names) {
+    const std::vector<std::string>&& counter_names) {
```

Comment:
At the risk of showing my incompetence, could you explain these changes from `const&` to `const&&`?  Const rvalue references are pretty uncommon in my experience, and I'd be surprised if Abseil was inviting their usage.  At a minimum, a comment would be welcome.

Reply:
Good catch, it's not needed.